### PR TITLE
Update enable-cloud-protection-microsoft-defender-antivirus.md

### PR DESCRIPTION
**Pull request by msbemba**
Removed the method to use intune to turn on cloud protection.

Fixes# https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10500

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/enable-cloud-protection-microsoft-defender-antivirus.md [hunk 1]
```diff
@@ -47,21 +47,6 @@ For more information about the specific network-connectivity requirements to ens
...
-
-3. Select the **Device restrictions** profile type you want to configure. If you need to create a new **Device restrictions** profile type, see [Configure device restriction settings in Microsoft Intune](/intune/device-restrictions-configure).
-
-4. In the **Properties** section, select **Edit** next to **Configuration settings** and then select **Microsoft Defender Antivirus**.
-
-5. On the **Cloud-delivered protection** switch, select **Enable**.
-
-6. In the **Prompt users before sample submission** dropdown, select **Send all samples automatically**.
-
-For more information about Intune device profiles, including how to create and configure their settings, see [What are Microsoft Intune device profiles?](/intune/device-profiles)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/enable-cloud-protection-microsoft-defender-antivirus.md]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@denisebmsft Please merge this PR. Thanks

**Review by denisebmsft — APPROVED** [review]
Thank you @msbemba

**Pull request merged**